server/lease/lessor.go

```diff
@@ -669,6 +669,9 @@ func (le *lessor) checkpointScheduledLeases() {
 		if len(cps) < maxLeaseCheckpointBatchSize {
 			return
 		}
+		// Clean up the cps, otherwise it will continue to schedule
+		// checkpoints even after it steps down the leader position.
+		cps = []*pb.LeaseCheckpoint{}
```

Comment:
Is there a reason to not make `cps` a local variable within the loop, so declare it in line 660 instead of 656 ?

Reply:
Ah:), good catch. Thanks & resolved.